docs/spec/openapi.yml

```diff
@@ -72,6 +72,85 @@ paths:
           application/json:
             schema:
               $ref: '#/components/schemas/UserCredentials'
+  '/permissions':
+    get:
+      tags:
+        - Authentication & Authorization
+      summary: Get user permissions
+      description: |
+        This API returns a list of permissions for the user that is currently logged in.
+        
+        *Example:*
+        Assume the following RBAC policy, and users `alice` and `bob:
+        ```
+        p, adminrole:role, namespaces, read, *
+        p, adminrole:role, database-engines, *, */*
+        p, adminrole:role, database-clusters, *, */*
+        p, bob, database-clusters, *, */*
+
+        g, alice, adminrole:role
+        ```
+
+        The API will return the following permissions for `alice`:
+        ```
+        {
+          "permissions": [
+            [
+                "adminrole:role",
+                "namespaces",
+                "read",
+                "*"
+            ],
+            [
+                "adminrole:role",
+                "database-engines",
+                "*",
+                "*/*"
+            ],
+            [
+                "adminrole:role",
+                "database-clusters",
+                "*",
+                "*/*"
+            ]
+          ]
+        }
```

Comment:
Can the FE work directly with this format? Will they need to convert it somehow?

Reply:
Yes, confirmed with @dianabirs that this should work